### ISSUE TITLE
Error while executing the npm install and npm link task

### ISSUE THREAD

**Issue by mergebandit** [transitive-bullshit/react-modern-library-cli#1]
The final step - `running npm install and npm link` is failing.

I'll look into this time permitting, but wanted to log it here. 

```
✖ Running npm install and npm link
{ Error: Command failed: /bin/sh -c npm


Usage: npm <command>

where <command> is one of:
    access, adduser, bin, bugs, c, cache, completion, config,
    ddp, dedupe, deprecate, dist-tag, docs, doctor, edit,
    explore, get, help, help-search, i, init, install,
    install-test, it, link, list, ln, login, logout, ls,
    outdated, owner, pack, ping, prefix, prune, publish, rb,
    rebuild, repo, restart, root, run, run-script, s, se,
    search, set, shrinkwrap, star, stars, start, stop, t, team,
    test, tst, un, uninstall, unpublish, unstar, up, update, v,
    version, view, whoami

npm <command> -h     quick help on <command>
npm -l           display full usage info
npm help <term>  search for help on <term>
npm help npm     involved overview

Specify configs in the ini-formatted file:
    /Users/scottsilvi/.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@5.3.0 /Users/scottsilvi/.nvm/versions/node/v8.5.0/lib/node_modules/npm

    at makeError (/Users/scottsilvi/.nvm/versions/node/v8.5.0/lib/node_modules/react-modern-library-cli/node_modules/execa/index.js:169:9)
    at Promise.all.then.arr (/Users/scottsilvi/.nvm/versions/node/v8.5.0/lib/node_modules/react-modern-library-cli/node_modules/execa/index.js:274:16)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  code: 1,
  stdout: '\nUsage: npm <command>\n\nwhere <command> is one of:\n    access, adduser, bin, bugs, c, cache, completion, config,\n    ddp, dedupe, deprecate, dist-tag, docs, doctor, edit,\n explore, get, help, help-search, i, init, install,\n    install-test, it, link, list, ln, login, logout, ls,\n    outdated, owner, pack, ping, prefix, prune, publish, rb,\n    rebuild, repo, restart, root, run, run-script, s, se,\n    search, set, shrinkwrap, star, stars, start, stop, t, team,\n    test, tst, un, uninstall, unpublish, unstar, up, update, v,\n    version, view, whoami\n\nnpm <command> -h     quick help on <command>\nnpm -l           display full usage info\nnpm help <term>  search for help on <term>\nnpm help npm     involved overview\n\nSpecify configs in the ini-formatted file:\n    /Users/scottsilvi/.npmrc\nor on the command line via: npm <command> --key value\nConfig info can be viewed via: npm help config\n\nnpm@5.3.0 /Users/scottsilvi/.nvm/versions/node/v8.5.0/lib/node_modules/npm\n',
  stderr: '',
  failed: true,
  signal: null,
  cmd: '/bin/sh -c npm',
  timedOut: false,
  killed: false }
```

**Comment by transitive-bullshit** [comment]
@mergebandit good catch; I didn't really test with npm.

Just pushed a fix that works locally for me as `v1.4.2`.  Let me know if that fixes your issue.

**Comment by mergebandit** [comment]
Works w/ `1.4.2`.
